### PR TITLE
Updated copy_to_glacier to use new Cumulus File Schema

### PR DESCRIPTION
**Pull request by AndrewDorn**
## Summary of Changes

Switched to new format for copy_to_glacier input

Addresses [ORCA-287: Copy to Glacier Input for files needs modification](https://bugs.earthdata.nasa.gov/browse/ORCA-287)

## Changes

* Modified input schema and code.
* Output format unchanged.

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Unit tests pass locally.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] CHANGELOG.md
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
    - [x] Unit tests
- [x] New and existing unit tests pass locally with my changes
    - [x] Automated tests passing (if not fork)
- [x] Any dependent changes have been merged and published in downstream modules
- [x] My code has passed security scanning
    - [x] Snyk
    - [x] git-secrets

### CODE REVIEW

**Review by rizbi-hassan — COMMENTED** [review]
is this going to create conflicts with orca-230?

**Comment by AndrewDorn** [comment]
> is this going to create conflicts with orca-230?

Yes. Unfortunately, Cumulus seems to be indicating that they cannot complete the changes required for that PR, and they need a new version of Orca that adapts to the schema changes they made to files.

**Review by rizbi-hassan — APPROVED** [review]

File: tasks/copy_to_glacier/copy_to_glacier.py [hunk 1]
```diff
@@ -124,24 +130,39 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
...
                                          multipart_chunksize_mb=multipart_chunksize_mb)
-            copied_file_urls.append(file['filename'])
-            print(f"Copied {source_filepath} into glacier storage bucket {default_bucket}.")
+            copied_file_urls.append(file_source_uri)
+            print(f"Copied {file_filepath} into glacier storage bucket {default_bucket}.")
 
     return {'granules': granules_list, 'copied_to_glacier': copied_file_urls}
 
 
+def switch_file_to_output_format(file: Dict[str, Any]) -> None:
```
Comment (bhazuka) [inline tasks/copy_to_glacier/copy_to_glacier.py:156]: Not sure the purpose of this. Wouldn't we follow the file schema Cumulus has put together?

File: tasks/copy_to_glacier/copy_to_glacier.py [hunk 2]
```diff
@@ -124,24 +130,39 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
...
         for file in granule['files']:
-            source_name = file['name']
-            source_filepath = file['filepath']
-            if should_exclude_files_type(source_name, exclude_file_types):
+            file_name = file[FILE_FILENAME_KEY]
+            file_filepath = file[FILE_FILEPATH_KEY]
+            file_bucket = file[FILE_BUCKET_KEY]
+            file_source_uri = file[FILE_SOURCE_URI_KEY]
+
+            switch_file_to_output_format(file)
```
[the remark this answers is not in the document]
Comment (AndrewDorn) [reply tasks/copy_to_glacier/copy_to_glacier.py]: Task just specified input, so I thought I needed to preserve output. Fixed.

File: tasks/copy_to_glacier/copy_to_glacier.py (hunk 1 above)
Comment (AndrewDorn) [reply tasks/copy_to_glacier/copy_to_glacier.py]: Task just specified input, so I thought I needed to preserve output. Fixed.

File: tasks/copy_to_glacier/schemas/output.json [hunk 3]
```diff
@@ -24,19 +24,19 @@
               "properties": {
                 "name": {
                   "type": "string",
-                  "description": "The name and extension of the file."
+                  "description": "Name of file (e.g. file.txt)."
```
[the remark this answers is not in the document]
Comment (AndrewDorn) [reply tasks/copy_to_glacier/schemas/output.json:27]: Task just specified input, so I thought I needed to preserve output. Fixed.

File: website/docs/developer/deployment-guide/deploying-from-windows.mdx [hunk 4]
```diff
@@ -62,7 +62,10 @@ This application will be used in future steps to authenticate users.
   VPC and Subnets are created by [NGAP](https://wiki.earthdata.nasa.gov/display/ESKB/AWS+Services+Approval+Status+Page#AWSServicesApprovalStatusPage-VPC).
   It is recommended you copy values from an existing deployment setup.
   :::
-- Go to [this repo](https://git.earthdata.nasa.gov/projects/ORCA/repos/cumulus-orca-template-deploy/browse) and clone it to your machine.
+- Go to [this repo](https://git.earthdata.nasa.gov/projects/ORCA/repos/cumulus-orca-deploy-template/browse) and clone it to your machine.
+:::warning
+The master branch is non-functional. These instructions have been tested with v9.4.0-v3.0.1
```
Comment (bhazuka) [inline website/docs/developer/deployment-guide/deploying-from-windows.mdx:67]: Actually the master branch is the fork of the Cumulus Template Repo located here https://github.com/nasa/cumulus-template-deploy and is used to help keep us in sync with any changes to the master template from Cumulus.

**Review by bhazuka — CHANGES_REQUESTED** [review]
Minor update needed to docs but overall looks good

File: website/docs/developer/deployment-guide/deploying-from-windows.mdx (hunk 4 above)
Comment (AndrewDorn) [reply website/docs/developer/deployment-guide/deploying-from-windows.mdx:67]: Presently the master branch points to non-existent Cumulus version 9.0.0. I can modify the warning, but I worry that it isn't maintained as it should be.

**Review by bhazuka — APPROVED** [review]

File: website/docs/developer/deployment-guide/deploying-from-windows.mdx (hunk 4 above)
Comment (bhazuka) [reply website/docs/developer/deployment-guide/deploying-from-windows.mdx]: That is true, Cumulus is not great at maintaining this.

**Pull request merged**